certmanager/util.go

```diff
@@ -33,12 +33,16 @@ type GcpTokenFetcher struct {
 	// aud is the unique URI agreed upon by both the instance and the system verifying the instance's identity.
 	// For more info: https://cloud.google.com/compute/docs/instances/verifying-instance-identity
 	aud            string
-	serviceAccount string
 }
 
 func (fetcher *GcpTokenFetcher) getTokenURI() string {
 	// The GCE metadata service URI to get identity token.
-	return "instance/service-accounts/" + fetcher.serviceAccount + "/identity?audience=" + fetcher.aud
+	return "instance/service-accounts/default/identity?audience=" + fetcher.aud
+}
+
+func (fetcher *GcpTokenFetcher) getServiceAccount() (string, error) {
```

Comment:
This is a private method but is not called by anyone. Consider adding it when it's actually needed?